endpoint/post.py

```diff
@@ -98,14 +102,18 @@ def on_delete(self, req, resp, id):
         try:
             user = req.context['user']
 
-            if (not user.is_logged_in()) or (not user.is_org()):
+            if (not user.is_logged_in()):
                 resp.status = falcon.HTTP_400
                 return
 
             post = session.query(model.Post).get(id)
             if post is None:
                 resp.status = falcon.HTTP_404
                 return
+            
+            if (not user.is_org() and post.author != user.id):
+                resp.status = falcon.HTTP_400
```

Comment:
please use [HTTP Code 403 (Forbidden)](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#4xx_client_errors), the code 400 was used by legacy backend code and I would like to change it to the newer format :slightly_smiling_face: 

```suggestion
                resp.status = falcon.HTTP_403
```